core/src/test/java/org/apache/iceberg/avro/TestReadProjection.java

```diff
@@ -221,10 +221,10 @@ public void testNestedStructProjection() throws Exception {
     );
 
     projected = writeAndRead("latitude_only", writeSchema, latOnly, record);
-    projectedLocation = (Record) projected.get("location");
-    Assert.assertNull("Should not project id", projected.get("id"));
+    Record projectedLocation = (Record) projected.get("location");
+    AssertHelpers.assertEmptyAvroField(projected, "id");
     Assert.assertNotNull("Should project location", projected.get("location"));
-    Assert.assertNull("Should not project longitude", projectedLocation.get("long"));
+    AssertHelpers.assertEmptyAvroField(projected, "long");
```

Comment:
I think this should still use `projectedLocation`. Looks like it might just be a typo.

Reply:
Good catch, probably a copy-paste!